samples/wifi/scan/src/main.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief WiFi scan sample
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(scan, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <nrfx_clock.h>
+#include <zephyr/zephyr.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/init.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
+
+#include "net_private.h"
+
+#define WIFI_SHELL_MODULE "wifi"
+
+#define WIFI_SHELL_MGMT_EVENTS (NET_EVENT_WIFI_SCAN_RESULT |		\
+				NET_EVENT_WIFI_SCAN_DONE)
+
+static uint32_t scan_result;
+
+static struct net_mgmt_event_callback wifi_shell_mgmt_cb;
+
+static void handle_wifi_scan_result(struct net_mgmt_event_callback *cb)
+{
+	const struct wifi_scan_result *entry =
+		(const struct wifi_scan_result *)cb->info;
+	uint8_t mac_string_buf[sizeof("xx:xx:xx:xx:xx:xx")];
+
+	scan_result++;
+
+	if (scan_result == 1U) {
+		printk("%-4s | %-32s %-5s | %-4s | %-4s | %-5s | %s\n",
+				"Num", "SSID", "(len)", "Chan", "RSSI", "Security", "BSSID");
```

Comment:
Please fix alignments across the file. It's preferred to align with the opening bracket for multi-line function calls/declarations.